app/utils/interfaces.ts

```diff
@@ -15,3 +15,24 @@ export interface UpdateEvent {
   start: stringOrDate;
   end: stringOrDate;
 }
+
+export interface Event {
+  id: number;
+  name: string;
+  description: string;
+  location: string;
+  startTime: string;
+  endTime: string;
+  ownerId: number;
+  eventTypeId: number;
+  calendarId: number;
+  isDeleted: boolean;
+  Attendees: {
+    attendee: {
+      email: string;
+    };
+  }[];
+  EventType: {
+    name: string;
+  };
+}
```

Comment:
why do we have some words starting with a capital letter and others not?

Reply:
I have fixed that, thanks yash for pointing out.